tests/test_objective_funs.py

```diff
@@ -569,6 +572,66 @@ def test(eq):
         test(get("DSHAPE"))
         test(get("HELIOTRON"))
 
+    @pytest.mark.unit
+    def test_coil_length(self):
+        """Tests coil length."""
+
+        def test(coil):
+            obj = CoilLength(coil, weight=1)
+            obj.build()
+            f = obj.compute(params=coil.params_dict)
+            np.testing.assert_allclose(f, 2 * np.pi, rtol=1e-8)
+
+        coil = FourierPlanarCoil(r_n=1)
+        coils = CoilSet.linspaced_linear(coil, n=4)
+        mixed_coils = MixedCoilSet.linspaced_linear(coil, n=4)
+        nested_coils = MixedCoilSet.from_symmetry(mixed_coils, NFP=4)
+
+        test(coil)
+        test(coils)
+        test(mixed_coils)
+        test(nested_coils)
+
+    @pytest.mark.unit
+    def test_coil_curvature(self):
+        """Tests coil curvature."""
+
+        def test(coil):
+            obj = CoilCurvature(coil, weight=1)
+            obj.build()
+            f = obj.compute(params=coil.params_dict)
+            np.testing.assert_allclose(f, 1 / 2, rtol=1e-8)
```

Comment:
see above about checking output size to make sure we didn't make any mistakes with the tree flatten/map stuff